src/components/Tab.tsx

```diff
@@ -1,48 +1,63 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
It is not needed, as this is warn-only rule. [The check will pass anyways](https://github.com/pluralsh/design-system/runs/7419709333?check_suite_focus=true#step:6:7).